src/EasyLiquidator.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.13;
+
+import "yield-utils-v2/token/IERC20.sol";
+import "yield-utils-v2/token/TransferHelper.sol";
+import "./interfaces/IERC3156FlashBorrower.sol";
+import "./interfaces/IERC3156FlashLender.sol";
+import "./interfaces/ISimpleSwap.sol";
+import "./interfaces/ICollateralizedVault.sol";
+
+
+contract EasyLiquidator is IERC3156FlashBorrower {
+
+    using TransferHelper for IERC20;
+
+    enum Action {NORMAL, OTHER}
+
+    IERC3156FlashLender public immutable lDAI;
+    ///IERC3156FlashLender public immutable lUSDC;
+    
+    ISimpleSwap public immutable sDAI;
+   /// ISimpleSwap public immutable sUSDC;
+
+    /// @notice contract targeted for liquidations, can carry multiple user vaults, deposits in WETH, debts in DAI and USDC
+    ICollateralizedVault public vault;
+
+    IERC20 public immutable WETH;
```

Comment:
We use lowerSnakeCase for state variables, but this one is immutable, so I guess that allcaps is alright.